frontend/tsconfig.json

```diff
@@ -1,6 +1,9 @@
 {
   "extends": "../tsconfig.json",
-  "include": ["src"],
+  "include": [
+    "src",
+    "../shared/src/packages/users/libs/types/delete-account-form-payload.type.ts"
```

Comment:
What is the purpose for it?

Reply:
@v0ldemar01 I don't recall manually adding that particular configuration. While I did unmount the Docker container and delete the database when switching branches, I don't remember modifying the tsconfig.json file. I will delete it.